tests/test_wave_01.py

```diff
@@ -1,10 +1,11 @@
+from operator import ge
```

Comment:
Looks like an extraneous auto-import? These shouldn't be committed but rather deleted before committing. I recommend turning off the auto-import feature in VS Code.